app/src/components/main/markdown/markdownEditor/markdownEditor.scss

```diff
@@ -113,6 +113,10 @@
     .CodeMirror {
       height: 70px;
       min-height: 50px;
+
+      &-placeholder {
+        font-size: 14px;
+      }
```

Comment:
It looks like issue affect only mode-dark view, so maybe better move this part to `mode-dark` class?